tests/e2e/watch_delay_test.go

```diff
@@ -166,6 +174,266 @@ func TestWatchDelayForEvent(t *testing.T) {
 	}
 }
 
+var watchKeyPrefix = "/registry/pods/"
+var watchCacheWatchKeyPrefix = "/registry/pod"
+
+type randomStringAlphabet string
+
+func (a randomStringAlphabet) makeString(minLen, maxLen int) string {
+	n := minLen
+	if minLen < maxLen {
+		n += rand.Intn(maxLen - minLen)
+	}
+	var s string
+	for i := 0; i < n; i++ {
+		s += string(a[rand.Intn(len(a))])
+	}
+	return s
+}
+
+var randomStringMaker = randomStringAlphabet("abcdefghijklmnopqrstuvwxyz0123456789")
+
+func TestWatchDelayOnStreamMultiplex(t *testing.T) {
+	e2e.BeforeTest(t)
+	clus, err := e2e.NewEtcdProcessCluster(t, &e2e.EtcdProcessClusterConfig{ClusterSize: 1, LogLevel: "info"})
```

Comment:
can reproduce it as well when `ClientHttpSeparate: true`. I was thinking that CPU resource or networking bandwitdh might impact this.

Reply:
Thanks for confirming.